common/smoke-test/Initialize-SmokeTests.ps1

```diff
@@ -90,20 +90,33 @@ function Set-EnvironmentVariables {
 
 function New-DeployManifest {
   Write-Verbose "Detecting samples..."
-  $javascriptSamples = (Get-ChildItem -Path "$repoRoot/sdk/$ServiceDirectory/*/samples/javascript/" -Directory
-    | Where-Object { Test-Path "$_/package.json" })
+  $packageDir = Get-ChildItem -Directory -Path "$repoRoot/sdk/$ServiceDirectory/*"
+
+  $javascriptSamples = @()
+  $packageDir.ForEach{
```

Comment:
Minor but I was suggesting a foreach loop here: (i.e. `foreach ($package in $packageDir) {`).